CHANGELOG.md

```diff
@@ -16,6 +16,9 @@ We are excited to be introducing new time-series analytical functions, advanced
 
 This release adds code under a new license, LICENSE_TIMESCALE. This code can be found in `tsl`.
 
+**For this release only**, you will need to restart the database before running
```

Comment:
Missing period, probably also skip "only", and just make :  For this release, you ...

(given that there could be future ones that need it as well.)